src/plugins/console/public/application/hooks/use_send_current_request_to_es/send_request_to_es.ts

```diff
@@ -6,6 +6,8 @@
  * Side Public License, v 1.
  */
 
+import { HttpSetup } from 'kibana/public';
+import _ from 'lodash';
```

Comment:
please don't do imports in that way. 
Legal way: `import { isUndefined } from 'lodash';`
But honestly not sure that we need lodash here at all